modules/openapi-generator/src/main/resources/aspnetcore/3.0/enumClass.mustache

```diff
@@ -5,8 +5,8 @@
         {{#description}}
         /// <value>{{{.}}}</value>
         {{/description}}
-        {{#allowableValues}}{{#enumVars}}{{#-first}}{{#isString}}[TypeConverter(typeof(CustomEnumConverter<{{datatypeWithEnum}}{{^datatypeWithEnum}}{{classname}}{{/datatypeWithEnum}}>))]
-        [JsonConverter(typeof(Newtonsoft.Json.Converters.StringEnumConverter))]{{/isString}}{{/-first}}{{/enumVars}}{{/allowableValues}}
+        {{#allowableValues}}{{#enumVars}}{{#-first}}{{#isString}}{{#useNewtonsoft}}[TypeConverter(typeof(CustomEnumConverter<{{datatypeWithEnum}}{{^datatypeWithEnum}}{{classname}}{{/datatypeWithEnum}}>))]
```

Comment:
Here is where I'm not sure how to handle without newtonsoft. Right now, I simply remove it completely and it compiles and all test pass. Not sure if it has other consequences, so feedback and confirmation would be appreciated.